neuraxle/steps/util.py

```diff
@@ -34,8 +34,10 @@
 from typing import List, Any
 
 from neuraxle.base import BaseStep, NonFittableMixin, NonTransformableMixin, MetaStepMixin, DataContainer
-from neuraxle.pipeline import PipelineSaver
+from neuraxle.base import TruncableSteps, \
+    NamedTupleList
```

Comment:
This could be in 1 line. Not important you can resolve this without fixing it because it works.